public/assets/transition-page/styles/transition-page.css

```diff
@@ -129,7 +130,7 @@ img {
   margin-left: 5px;
 }
 
-@media only screen and (min-width: 600px) {
+@media only screen and (min-width: 960px) {
```

Comment:
can we use more canonical breakpoints such as one of [these](https://ricostacruz.com/til/css-media-query-breakpoints)?

Reply:
Sound great, but doing that might affect consistency a little as the 600px / 960px values are referenced from default MUI breakpoints that we are using in our frontend React app.

Reply:
Ok. I see that these are the [defaults](https://material-ui.com/customization/breakpoints/) in MUI